README.md

```diff
@@ -223,6 +237,22 @@ Then edit the JSON so that the key `ssl/openssl/base_conf_path` has a
 value that is the path to the OpenSSL config file (typically this
 will be `C:\\OpenSSL-Win32\\bin\\openssl.cfg`).
 
+After setting `%Path%` variables, be sure to exit your current cmd
+prompts and relaunch.
+
+Alternatively, you may also use
+[Bash on Ubuntu on Windows](https://msdn.microsoft.com/en-us/commandline/wsl/about)
+in the Windows 10 Anniversary Update build, then access your windows
+partition from there to launch `wpt` commands.
+
+The first time you run the tests in Microsoft Edge, you will be asked to
```

Comment:
Want to make a patch to set that automatically? Does that require Admin permissions?